shared/src/validation-schemas/episode/episode.validation-schema.ts

```diff
@@ -30,6 +30,8 @@ const episode = Joi.object<EpisodeCreatePayload>({
       'number.required': EpisodeValidationMessage.PODCAST_ID_REQUIRE,
       'number.integer': EpisodeValidationMessage.PODCAST_ID_NUMBER_FORMAT,
     }),
+  [EpisodeCreatePayloadKey.TYPE]: Joi.string()
+    .valid(...Object.values(EpisodeTypeKey))
```

Comment:
```suggestion
    .valid(Object.values(EpisodeTypeKey))
```

Reply:
Without this destructuring "...Object.values" application crashes